### DISCUSSION TITLE
xs, sm, md, lg, xl, xxl döşenek adlarının Türkçesi ne olmalı?

### DISCUSSION THREAD

**Discussion by volkantash** [data-deveb/deveb.css#3] [Soru-Yanıt]
Selâmun aleyküm. Arkadaşlar döşenek (layout) için İngilizcede birkaç ad kullanılıyor. Size bu adların neler olduğundan şöyle söz edebilirim.

xs: extra small
sm: small
md: medium
lg: large
xl: extra large
xxl: extra extra large

Peki bunların her birine Türkçe karşılık için hangi sözcüğü önerirdiniz?

**Comment by yankikucuk** [comment]
xs: küçücük
sm: küçük
md: ortanca
lg: büyük
xl: çok büyük
xxl: kocaman

**Reply by volkantash** [reply]
küçük, küçücük ikilisinde sizinle oydaşım. 💯 

**medium** ile **middle** sözcüklerine hep "orta" demişiz. Orada "ortanca" demek yerli yerinde olur mu diye tartmamız gerekecek. 

**Large** için **büyük** der isek; **big** için ne diyeceğiz? 

Bu yüzden biraz daha üzerinde çalışmayı öneriyorum.
